### PR TITLE
release SDK 6 0.9.0

### PR DESCRIPTION
**Pull request by denisbedin**
https://docs.microej.com/en/feature-microej-sdk-6_0.9.0/SDK6UserGuide/sdk6ChangeLog.html

### CODE REVIEW

File: SDK6UserGuide/sdk6MigrationNotes.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.. _sdk_6_migration_notes:
+
+Migration Notes
+---------------
+
+-------------------
+From 0.8.0 to 0.9.0
+-------------------
+
+This section applies if MicroEJ SDK 6 ``0.9.0`` is used on a project that was created using MicroEJ SDK 6 ``0.8.0``.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6MigrationNotes.rst:10]:
```suggestion
This section applies if MicroEJ SDK 6 ``0.9.0`` is used on a project that was created using MicroEJ SDK 6 ``0.8.0`` or lower.
```

File: SDK6UserGuide/sdk6MigrationNotes.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Merge of the veePortDirs and veePortFiles properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The build properties ``veePortDirs`` and ``veePortFiles`` have been merged into a single property ``veePortPaths``.
+To define a local VEE Port, set the build property ``veePortPaths`` in the ``microej`` configuration block 
+to the path of the VEE Port file (``.zip`` or ``.vde``) or to the source folder of the VEE Port::
+
+   microej {
+     veePortPaths = listOf("C:\\path\\to\\my\\veePort\\file.zip")
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6MigrationNotes.rst:20]: I would use the folder example instead since it is the most common case.

**Review by tdelhomenie-microej — CHANGES_REQUESTED** [review]

File: SDK6UserGuide/sdk6MigrationNotes.rst (hunk 1 above)
Comment (denisbedin) [reply SDK6UserGuide/sdk6MigrationNotes.rst]: Done.

File: SDK6UserGuide/sdk6MigrationNotes.rst (hunk 2 above)
Comment (denisbedin) [reply SDK6UserGuide/sdk6MigrationNotes.rst]: Done.

**Review by tdelhomenie-microej — APPROVED** [review]

**Pull request merged**